pyproject.toml

```diff
@@ -10,7 +10,7 @@ requires = ["scikit-build-core"]
 build-backend = "scikit_build_core.build"
 
 [project]
-name = "pynvjitlink-cu12"
+name = "pynvjitlink"
```

Comment:
I changed this so that the package name would align for conda. I modified `build_wheel.sh` and `test_wheel.sh` accordingly, so they add a `-cu12` suffix. This matches our "normal" approach for RAPIDS wheels.